tmt/convert.py

```diff
@@ -138,7 +138,7 @@ def html_to_markdown(html):
         import html2text
         md_handler = html2text.HTML2Text()
     except ImportError:
-        raise ConvertError("Install html2text to import manual tests.")
+        raise ConvertError("Install tmt-test-convert to import manual tests.")
```

Comment:
It is not only about manual tests. https://github.com/psss/tmt/blob/af7da979960dcb76357acba28db5a1633c34e7dc/tmt/convert.py#L406

Reply:
typical, I forgot to change the one reason this PR was created for...
fixed